.github/dependabot.yml

```diff
@@ -1,5 +1,11 @@
 version: 2
 updates:
+- package-ecosystem: docker
```

Comment:
Wait do we need to set reviewers here as well?

Reply:
Ah yes, thanks. updated